docs/guide/persistence_cache.md

```diff
@@ -222,6 +222,21 @@ stash:
     The regular `php app/console cache:clear` command does not clear Redis persistence cache.
     To clear it, use the console command shipped with Redis: `redis-cli flushall`.
 
+!!! tip
+
+    If you use the Redis cache driver and encounter problems with memory consumption,
+    you can use the following (non-SiteAccess-aware) settings:
+
+    ``` yaml
+    ezpublish:
+        stash_cache:
+            igbinary: true
+            lzf: true
+    ```
+
+    - `ezpublish.stash_cache.igbinary` enables you to use the `igbinary` library to serialize objects stored in cache.
+    - `ezpublish.stash_cache.lzf` enables you to use the `LZF` library to compress serialized objects stored in cache.
+
```

Comment:
Please, add also a note that clearing cache and purge of Redis content is necessary after changing this configuration.

Reply:
Do we have an example of that somewhere perhaps? for instance using redis cli.